src/libslic3r/PrintConfig.cpp

```diff
@@ -3073,20 +3073,6 @@ def = this->add("filament_loading_speed", coFloats);
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionEnumsGeneric{ ZHopType::zhtNormal });
 
-    def = this->add("retract_lift_above", coFloats);
```

Comment:
Nice catch.
Can we remove the other one since these descriptions are well translated into other languages by various contributors already?